packages/documentation/src/stories/components/alert/web-component/alert-dismiss.sample.ts

```diff
@@ -0,0 +1,3 @@
+const alert = document.getElementById('alertId') as HTMLPostAlertElement;
+
+alert.dismiss().then(() => {});
```

Comment:
```suggestion
alert.dismiss()
```
Nitpick: I'm not sure that it is necessary to show the then.

Reply:
It is to highlight that the `.dimiss()` method returns a promise